back/cmd/openbsrv/main.go

```diff
@@ -7,17 +7,32 @@ import (
 	"path"
 
 	"github.com/codemodus/sigmon/v2"
+
+	"github.com/OpenEugene/openboard/back/internal/dbg"
+	"github.com/OpenEugene/openboard/back/internal/log"
 )
 
 func main() {
-	if err := run(); err != nil {
+	config := log.Config{
+		Err: log.Output{
+			Out:    os.Stderr,
+			Prefix: "[ERROR] ",
+		},
+		Inf: log.Output{
+			Out:    os.Stdout,
+			Prefix: "[INFO] ",
+		},
+	}
+	log := log.New(config)
+
+	if err := run(log); err != nil {
 		cmd := path.Base(os.Args[0])
-		fmt.Fprintf(os.Stderr, "%s: %s\n", cmd, err)
+		log.Error("%s: %s", cmd, err)
```

Comment:
Here I suggest fmt.Fprintf to stderr. This means that the log can be setup in the run function rather in the main itself.